.github/workflows/translation-updates-to-main-repo.yml

```diff
@@ -0,0 +1,144 @@
+          done
+          for file in *
+          do
+            for language in "${languages[@]}"
+            do po2md $file --po-files ../../../translation-files/contribute-pages/site-contribute_$language.po --save ../../$language/contribute/$file
+            git add ../../$language/contribute/$file
+            done
+          done
+      - name: Make Documentation pages
+        run: |
+          # top-level md files
+          cd _i18n/en/documentation
+          languages=(${{ env.languages }})
+          for language in "${languages[@]}"
+          do mkdir -p ../../$language/documentation
+          done
+          for file in *.md
+          do
+            for language in "${languages[@]}"
+            do po2md $file --po-files ../../../translation-files/documentation-pages/site-documentation_$language.po --save ../../$language/documentation/$file
+            git add ../../$language/documentation/$file
+            done
+          done
+          # sub-level files
+          documentation_groups=(automation general getting-started playback podcasters-hosters queue subscriptions)
+          for category in "${documentation_groups[@]}"
+          do
+            cd $category
+            languages=(${{ env.languages }})
+            for language in "${languages[@]}"
+            do mkdir -p ../../../$language/documentation/$category
+            done
+            for file in *
+            do
+              for language in "${languages[@]}"
+              do
+                po2md $file --po-files ../../../../translation-files/documentation-pages/site-documentation_$language.po --save ../../../$language/documentation/$category/$file
+              git add ../../../$language/documentation/$category/$file
+              done
+            done
+            cd ../
+          done
+      - name: Make General pages
+        run: |
+          cd _i18n/en/general
+          ls
+          languages=(${{ env.languages }})
+          for language in "${languages[@]}"
+          do mkdir -p ../../$language/general
+          done
+          for file in *
+          do
+            for language in "${languages[@]}"
+            do po2md $file --po-files ../../../translation-files/general-pages/site-general_$language.po --save ../../$language/general/$file
+            git add ../../$language/general/$file
+            done
+          done
+      - name: Stage yml files
+        run: |
+          languages=(${{ env.languages }})
+          for language in "${languages[@]}"
+          do cp translation-files/general-strings/$language.yml _i18n/$language.yml
+          git add _i18n/$language.yml
+          done
+      - name: Commit changes
+        run: |
+          git config user.name "AntennaPod Translations Bot"
+          git config user.email "<antennapod-bot@users.noreply.github.com>"
+          git status
+          git commit -m "Update website with new translations"
+          git push origin master
```

Comment:
This makes a commit on translation-files and then pushes master, I think. So if I read this correctly, it doesn't do anything.